### PR TITLE
Coin Change Problem

### PR DESCRIPTION
**Pull request by harsh7in**
issue #216 solved.

### CODE REVIEW

**Comment by atarax665** [comment]
@harsh-9in Make changes to the README file and please improve the readability of your code.

File: C++/dp/Coin_Change_Problem.cpp [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#include<bits/stdc++.h>
```
Comment (atarax665) [inline C++/dp/Coin_Change_Problem.cpp]: Add description of the problem.

**Review by atarax665 — CHANGES_REQUESTED** [review]

File: C++/dp/Coin_Change_Problem.cpp (hunk 1 above)
Comment (harsh7in) [reply C++/dp/Coin_Change_Problem.cpp]: Given a value V, if we want to make change for V cents, and we have infinite supply of each of C = { C1, C2, .. , Cm} valued coins, what is the minimum number of coins to make the change?
Examples:

Input: coins[] = {25, 10, 5}, V = 30
Output: Minimum 2 coins required
We can use one coin of 25 cents and one of 5 cents 

Input: coins[] = {9, 6, 5, 1}, V = 11
Output: Minimum 2 coins required
We can use one coin of 6 cents and 1 coin of 5 cents

**Comment by harsh7in** [comment]
@atarax665 Plz review the changes.

**Comment by atarax665** [comment]
@harsh-9in I have made all the required changes. Try to write clean and maintainable code. Btw Good Work!!

**Pull request merged**